### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by furkanygt1** [community/community#65825] [GitHub Education]
### Hi everyone,

I don't access student developer pack. My application approved but I don't access ony property. The site says my membership is reviewing. How can I solve?

**Comment by jesse-sealand** [comment]
How long ago did you submit your application and how long has it been since its been in 'review' status?

**Reply by furkanygt1** [reply]
I apply to students pack at 31 August. It's been 4 day.

**Reply by furkanygt1** [reply]
Thanks a lot my problem solved

**Comment by furkanygt1** [comment]
I was accessing my domain with github but site says "" The custom domain `www.crndanisiyorum.me` is already taken by another repository in your account. Check out https://docs.github.com/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site for information about how to remove this domain from the other repository.""  What should I do?
